ledger-core/network/gateway/base.go

```diff
@@ -204,6 +211,7 @@ func (g *Base) StartConsensus(ctx context.Context) error {
 	if network.OriginIsJoinAssistant(cert) {
 		// one of the nodes has to be in consensus.ReadyNetwork state,
 		// all other nodes has to be in consensus.Joiner
+		// TODO: fix Assistant node can't join existing network
```

Comment:
pls add JIRA to the backlog and mention it here

Reply:
PLAT-694